src/librustc_parse/parser/pat.rs

```diff
@@ -295,6 +295,21 @@ impl<'a> Parser<'a> {
             // A rest pattern `..`.
             self.bump(); // `..`
             PatKind::Rest
+        } else if self.check(&token::DotDotDot) && !self.is_pat_range_end_start(1) {
+            // A rest pattern `...`.
+            self.bump(); // `...`
+
+            // A ... is not expected here
+            self.struct_span_err(lo, "unexpected `...`")
+                .span_label(lo, "not expected here")
+                .span_suggestion(
+                    self.token.span,
+                    "Did you mean to use one fewer `.`?",
```

Comment:
```suggestion
                    "for a rest pattern, use `..` instead of `...`",
```
(Our style is to always start lowercase and avoid posing questions.)